tests/dummy/app/router.js

```diff
@@ -1,12 +1,14 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-const Router = Ember.Router.extend({
+const { Router } = Ember;
+
+const EmRouter = Router.extend({
```

Comment:
@kellyselden Can I make that change then? From which ember-cli is it already available? I've seen that ember-basic-dropdown is already using it, although it's still in beta.

https://github.com/cibernox/ember-basic-dropdown/blob/master/addon/components/basic-dropdown/content.js#L1